test/test_cpp_extensions_open_device_registration.py

```diff
@@ -53,46 +53,6 @@ def setUpClass(cls):
             verbose=True,
         )
 
-    def test_open_device_faketensor(self):
-        with torch._subclasses.fake_tensor.FakeTensorMode.push():
-            a = torch.empty(1, device="openreg")
-            b = torch.empty(1, device="openreg:0")
-            result = a + b  # noqa: F841
-
-    def test_open_device_named_tensor(self):
-        torch.empty([2, 3, 4, 5], device="openreg", names=["N", "C", "H", "W"])
```

Comment:
Note that named dimensions are slated to removal soon. So no need to spend too much time on these.

Reply:
Got it ,thank you.